### DISCUSSION TITLE
Sync data from Google Drive to external drive

### DISCUSSION THREAD

**Discussion by platypusrex256** [iterate-ch/cyberduck#15036] [Q&A]
I have a large 60TB+ GoogleDrive account and I'm struggling to download and sync that data to an external drive. For some reason, it appears that Mac OS Ventura won't allow Google Drive app to specify a folder for sync so you're forced to use the default boot drive.

Several people have suggested MountainDuck and I'm just worried that I'll run into the same issues here as the root of the problem might be with Ventura, not the software. Anybody here running Ventura having problems?

**Comment by platypusrex256** [comment]
Not feeling great about support here... I paid for a license against my best judgement. App feels extremely buggy.

**Reply by platypusrex256** [reply]
Where is the support?

**Comment by tatze96** [comment]
Mountain Duck isn't able to synchronize folders with each other. It is only able to synchronize files to the server and into a local cache for later offline usage while using the _Smart Synchronization_ mode.

You can set the synchronization cache to an external drive, but this won't allow you to access the files directly on that drive as those files are obfuscated, and you can only access them from within the mounted Mountain Duck drive.

For more information regarding the _Smart Synchronization_ mode, refer to our [wiki page](https://docs.cyberduck.io/mountainduck/connect/sync/).

You can use Cyberduck to manually synchronize a local folder with a remote folder and vice versa with the help of its [synchronization feature](https://docs.cyberduck.io/cyberduck/sync/) or try out one of the options stated within the [Other Solutions](https://docs.cyberduck.io/cyberduck/sync/#other-solutions) section of the same article.

Downloading items to any local location using Mountain Duck shouldn't cause issues.

**Reply by platypusrex256** [reply]
I don't know what any of that means. Can I not use this app to download my stuff from Google Drive? Whats the point of app if I can't download stuff?

**Reply by tatze96** [reply]
You can download your files from Google Drive with Mountain Duck.

**Comment by platypusrex256** [comment]
Okay got it...  I'm trying to use MountainDuck to connect to my Google Drive and download a bunch of data (maybe 10TB) but every time I go to connect, it crashes the app.

It was working for a while but then my target drive came disconnected and when I hooked it back up, it stopped working.

Finder says the drive has 100% space available and yet about half of it used... which is weird. And I can't see any files.

**Comment by tatze96** [comment]
Please send us the latest [crash report](https://docs.cyberduck.io/mountainduck/support/#crash-reports) for the mentioned crashes and a [debug log](https://docs.cyberduck.io/mountainduck/support/#debug-log) right after the empty file listing occurs again to support@mountainduck.io for further investigation. Ensure to add the ticket reference GH-15036 so we can match the tickets quickly.
